packages/ui/src/components/SearchMultiSelectInput/index.tsx

```diff
@@ -288,14 +288,25 @@ function SearchMultiSelectInput<
         },
         [name, onChange],
     );
+    const handleSelectAll = useCallback(
+        () => {
+            if (isNotDefined(options)) {
+                return;
+            }
 
+            const allValues = options.map(keySelector);
+            onChange(allValues, name);
+        },
+        [options, name, onChange, keySelector],
+    );
     return (
         <SelectInputContainer
             // eslint-disable-next-line react/jsx-props-no-spreading
             {...otherProps}
             name={name}
             options={realOptions}
             optionsPending={optionsPending}
+            selectedOptions={selectedOptions}
```

Comment:
Why was this line added? Was there previously a bug?

Reply:
No, it was already there.